packages/fast-components-class-name-contracts-msft/src/action-trigger/index.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * The class name contract for the action trigger component
+ */
+export interface ActionTriggerClassNameContract {
+    /**
+     * The root of the action trigger component
+     */
+    actionTrigger?: string;
```

Comment:
Should this be optional?

Reply:
Yeah, I see that... hey @nicholasrice why do we do it that way? Won't the root MSFT class _always_ be there in a styled component?

Reply:
I mean - technically, this only becomes styled once we style it. The component as it exists as a class in the .tsx file can technically be used without style.